api/src/main/java/io/opencensus/stats/AggregationData.java

```diff
@@ -417,6 +417,20 @@ public static DistributionData create(
     @AutoValue
     public abstract static class Exemplar {
 
+      /**
+       * Key for {@code TraceId} in the contextual information of an {@link Exemplar}.
```

Comment:
Probably you want to use hex lower case for both values.

Reply:
Sure, I added a note about the suggested encoding format on values.